packages/relayer/indexer/handle_event.go

```diff
@@ -57,27 +57,6 @@ func (svc *Service) handleEvent(
 		return errors.Wrap(err, "svc.processMessage")
 	}
 
-	// if the block number of the event is higher than the block we are processing,
-	// we can now consider that previous block processed. save it to the DB
-	// and bump the block number.
-	if raw.BlockNumber > svc.processingBlock.Height {
-		log.Infof("saving new latest processed block to DB: %v", raw.BlockNumber)
-
-		if err := svc.blockRepo.Save(relayer.SaveBlockOpts{
-			Height:    svc.processingBlock.Height,
```

Comment:
Just checked `setInitialProcessingBlockByMode` ( https://github.com/taikoxyz/taiko-mono/blob/main/packages/relayer/indexer/set_initial_processing_block_by_mode.go#L31 ) , seems when re-syncing L1 (especially eth mainnet) , the starting height should be [this genesis height](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/L1/LibData.sol#L54) but not zero?

Reply:
ah I wasn't aware of this. I can add that!

Reply:
I pushed up the changes, does it look right?